### PR TITLE
TRM-29103: idMapping for deleted UniProtKB entries does not show status

### PR DESCRIPTION
**Pull request by LeonardoGonzales**
idMapping for deleted UniProtKB entries does not return inactiveStatus

### CODE REVIEW

File: idmapping-rest/src/test/java/org/uniprot/api/idmapping/controller/UniProtKBIdMappingStreamControllerIT.java [hunk 1]
```diff
@@ -376,6 +381,36 @@ void streamCanReturnLineageData() throws Exception {
...
+                        .createAndPutJobInCache(
+                                UNIPROTKB_AC_ID_STR, UNIPROTKB_STR, "Q00001,I8FBX0");
+        ResultActions response =
+                performRequest(
+                        get(getIdMappingResultPath(), job.getJobId())
+                                .param("fields", "accession")
+                                .header(ACCEPT, APPLICATION_JSON_VALUE));
+        // then
+        response.andDo(print())
+                .andExpect(status().is(HttpStatus.OK.value()))
```
Comment (ahmadshadab) [inline idmapping-rest/src/test/java/org/uniprot/api/idmapping/controller/UniProtKBIdMappingStreamControllerIT.java:398]: Replace print method with log

File: idmapping-rest/src/test/java/org/uniprot/api/idmapping/controller/utils/AbstractJobOperation.java [hunk 2]
```diff
@@ -92,7 +97,7 @@ protected IdMappingJobRequest createRequest(String from, String to, String fromI
         return request;
     }
 
-    private String generateHash(IdMappingJobRequest request) {
+    private String generateHash() {
         return UUID.randomUUID().toString();
```
Comment (ahmadshadab) [inline idmapping-rest/src/test/java/org/uniprot/api/idmapping/controller/utils/AbstractJobOperation.java:101]: thanks

**Review by ahmadshadab — COMMENTED** [review]
looks good to me. 1-2 minor comments. But could you please add a test to check if the accession is neither in store nor deleted(in solr)?

**Comment by LeonardoGonzales** [comment]
This is not a valid use case. Because if the id is not in solr and is not in store it will be returned as unmappedId by PIR. 
Remember this is the result use case, all the ids sent were already verified by PIR.

**Review by ahmadshadab — APPROVED** [review]

**Pull request merged**